### ISSUE TITLE
About load pretrain model 

### ISSUE THREAD

**Issue by cjf9028** [google-research/bert#513]
Unable to open table file model/bert_model.ckpt.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?

tensorflow 1.12.0
cuda 9.0

how to load pretrain model

**Comment by yousraHazem** [comment]
I have a question, whose answer maybe closely related to yours; I am confused as to the parameter to provide for init_checkpoint while pretraining. README describes downloadable pretrained BERT files as follows:
```
Each .zip file contains three items:

- A TensorFlow checkpoint (bert_model.ckpt) containing the pre-trained weights (which is actually 3 files).
- A vocab file (vocab.txt) to map WordPiece to word id.
- A config file (bert_config.json) which specifies the hyperparameters of the model.
```

Then in the pretraining section we pass `--init_checkpoint=$BERT_BASE_DIR/bert_model.ckpt`, but it is already unraveled into 3 different files at download. 

Question is, are we required to pass them as one `.ckpt` file or just use `bert_model.ckpt.data-00000-of-00001`? And if we're required to pass one `.ckpt` file then how do make that file from the downloaded 3 files?

**Comment by vtrokhymenko** [comment]
did u understood how to do this?

**Comment by lotest** [comment]
Same error!
